storage/cloudspanner/map_storage.go

```diff
@@ -110,6 +112,11 @@ func (ms *mapStorage) SnapshotForTree(ctx context.Context, tree *trillian.Tree)
 	return ms.begin(ctx, tree, true, ms.ts.client.ReadOnlyTransaction())
 }
 
+// Layout is not implemented.
+func (ms *mapStorage) Layout(tree *trillian.Tree) (*tree.Layout, error) {
+	return nil, errors.New("not implemented")
```

Comment:
Does this (& similar stubs, below) mean that this PR removes cloudspanner support for the Map?

Reply:
Yes.

Reply:
This is awkward because of the 2 customers we need to support with the map, 1/2 of them want to use CloudSpanner. I've raised #2001 to allow this to move ahead.